mindmap/static/js/src/mindmapEdit.js

```diff
@@ -0,0 +1,17 @@
+/* Javascript for MindMapXBlock. */
+function MindMapXBlock(runtime, element) {
+
+    $(element).find('.save-button').bind('click', function() {
+        var handlerUrl = runtime.handlerUrl(element, 'studio_submit');
+        var data = {
+            display_name: $(element).find('input[name=mindmap_display_name]').val(),
+        };
+        $.post(handlerUrl, JSON.stringify(data)).done(function(response) {
+          window.location.reload(false);
+        });
+      });
+
+    $(element).find('.cancel-button').bind('click', function() {
```

Comment:
It's better if you use `.click` instead of `.bind` it would look like this:

```
$(element).find('.cancel-button').click(function() {
        runtime.notify('cancel', {});
});
```

Reply:
Done